01_Data/src/layers/sequelize/model/Authorization/AdditionalInfo.ts

```diff
@@ -3,17 +3,17 @@
 //
 // SPDX-License-Identifier: Apache 2.0
 
-import { OCPP2_0_1_Namespace, OCPP2_0_1 } from '@citrineos/base';
+import {OCPP2_0_1_Namespace, OCPP2_0_1, Namespace} from '@citrineos/base';
 import { BelongsToMany, Column, DataType, Model, Table } from 'sequelize-typescript';
 import { IdToken } from './IdToken';
 import { IdTokenAdditionalInfo } from './IdTokenAdditionalInfo';
 
 @Table
-export class AdditionalInfo extends Model implements OCPP2_0_1.AdditionalInfoType {
-  static readonly MODEL_NAME: string = OCPP2_0_1_Namespace.AdditionalInfoType;
+export class AdditionalInfo extends Model {
+  static readonly MODEL_NAME: string = Namespace.AdditionalInfoType;
 
   @BelongsToMany(() => IdToken, () => IdTokenAdditionalInfo)
-  declare additionalInfo?: OCPP2_0_1.IdTokenType[];
+  declare additionalInfo?: object[];
```

Comment:
Would be better to use the sequelize model here for this association.
```suggestion
  declare additionalInfo?: IdToken[];
```